daft/io/_csv.py

```diff
@@ -21,13 +21,14 @@
 def read_csv(
     path: Union[str, List[str]],
     schema_hints: Optional[Dict[str, DataType]] = None,
+    schema_override: Optional[Dict[str, DataType]] = None,
```

Comment:
Should we throw an error if both `schema_hints` and `schema_override` are provided?

Reply:
Yes, will add a check for this.